indexer/road_shields_parser.cpp

```diff
@@ -487,7 +507,7 @@ class FinlandRoadShieldParser : public NumericRoadShieldParser
                                                {40, 99, RoadShieldType::Generic_Orange},
                                                {100, 999, RoadShieldType::Generic_White},
                                                {1000, 9999, RoadShieldType::Generic_Blue},
-                                               {10000, 60000, RoadShieldType::Hidden}})
+                                               {10000, std::numeric_limits<uint16_t>::max(), RoadShieldType::Hidden}})
```

Comment:
Add kHighRoadNumber constant near NumericRoadShieldParser instead of boring std::numeric_limits<uint16_t>::max()?

Reply:
done